exchanges/hitbtc/hitbtc_types.go

```diff
@@ -184,6 +184,7 @@ type OrderHistoryResponse struct {
 	Type          string    `json:"type"`
 	TimeInForce   string    `json:"timeInForce"`
 	Price         float64   `json:"price,string"`
+	AvgPrice      float64   `json:"avgPrice,string"`
```

Comment:
Reference: https://api.hitbtc.com/v2#orders-history (they got v3 API now but v2 is the one using by gct)

avgPrice is a valid field